src/analysis/normed_space/normed_group_hom.lean

```diff
@@ -273,16 +286,27 @@ variables {f g}
 def id : normed_group_hom V V :=
 (add_monoid_hom.id V).mk_normed_group_hom 1 (by simp [le_refl])
 
-/-- The norm of the identity is at most `1`. It is in fact `1`, except when the space is trivial
-where it is `0`. It means that one can not do better than an inequality in general. -/
+/-- The norm of the identity is at most `1`. It is in fact `1`, except when the norm of every
+element vanishes, where it is `0`. (Since we are working with seminorms this can happen even if the
+space is non-trivial.) It means that one can not do better than an inequality in general. -/
 lemma norm_id_le : ∥(id : normed_group_hom V V)∥ ≤ 1 :=
 op_norm_le_bound _ zero_le_one (λx, by simp)
 
-/-- If a space is non-trivial, then the norm of the identity equals `1`. -/
-lemma norm_id [nontrivial V] : ∥(id : normed_group_hom V V)∥ = 1 :=
-le_antisymm norm_id_le $ let ⟨x, hx⟩ := exists_ne (0 : V) in
+/-- If the is an element with norm different from `0`, then the norm of the identity equals `1`.
+(Since we are working with seminorms supposing that the space is non-trivial is not enough.) -/
```

Comment:
```suggestion
/-- If there is an element with norm different from `0`, then the norm of the identity equals `1`.
(Since we are working with seminorms supposing that the space is non-trivial is not enough.) -/
```